packages/askui-nodejs/src/execution/dsl.ts

```diff
@@ -807,8 +805,7 @@ export class FluentFiltersOrRelations extends FluentFilters {
    * @return {FluentFilters}
    */
   below(
-    optionalIndex: number | undefined
-    = 0,
+    optionalIndex = 0,
   ): FluentFilters {
```

Comment:
below(undefined) is valid too , and it's the inference response when we request text below text to fluent api => text(undefined).below(undefined).text(undefined)